lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -297,7 +313,7 @@ def self.available_options
 
           FastlaneCore::ConfigItem.new(key: :app_name,
                                   env_name: "APPCENTER_APP_NAME",
-                               description: "App name as found in the App's URL in App Center. If there is no app with such name, you will be prompted to create one",
+                               description: "App name as found in the App's URL in App Center, if there is no app with such name, you will be prompted to create one",
```

Comment:
The period looks more grammatically correct.

Reply:
Right, this was meant to unify README and this (they should be copy-paste)